web/views/manifestation.py

```diff
@@ -0,0 +1,43 @@
+# TODO: remove this
+from collections import namedtuple
+
+from flask import request, Blueprint
+from flask_restful import Resource, Api
+
+from coalaip_bigchaindb.plugin import Plugin
+from coalaip import CoalaIp
+
+
+coalaip = CoalaIp(Plugin('http://localhost:9984/api/v1'))
+
+manifestation_views = Blueprint('manifestation_views', __name__)
+manifestation_api = Api(manifestation_views)
+
+
+class ManifestationApi(Resource):
+    def post(self):
+        data = request.get_json(force=True)
```

Comment:
well, if so, then maybe make sure there is a graceful error path for the user if he didnt provide one of the keys below?